Source/Private/ASDisplayNode+UIViewBridge.mm

```diff
@@ -207,7 +202,6 @@ - (ASCornerRoundingType)cornerRoundingType
 
 - (void)setCornerRoundingType:(ASCornerRoundingType)newRoundingType
 {
-  ASDN::MutexLocker l(__instanceLock__);
   [self updateCornerRoundingWithType:newRoundingType cornerRadius:_cornerRadius];
```

Comment:
Isn't this public? Think we should read `self.cornerRadius`?

Reply:
yeap, nice catch!